commit d6fa33f6b64926d5c6907e358eb2bb92a39933cd (refs/changes/13/5613/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-02-19T16:03:38-08:00 (8 months ago)
    
    TRITON-1242 Create DAPI filter to allow filtering out virtual servers

diff --git a/package.json b/package.json
index 9fd8b77..dd865e5 100644
--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "strsplit": "1.0.0",
     "tape": "^4.4.0",
     "trace-event": "1.2.0",
-    "triton-tags": "1.2.2",
+    "triton-tags": "1.3.0",
     "ufds": "1.2.0",
     "vasync": "2.1.0",
     "verror": "1.9.0",
